database_schema.md

```diff
@@ -18,6 +18,8 @@ event_id: {
     thumbnail_hover_uri: str
     video_uri: str
     transcript_id: str
+    transcript_file_id: str
+    transcript_file_uri: str
```

Comment:
I actually think we may want to take out all the transcript details from the event collection. Any ideas when we would need it for "fast ui access"?

Reply:
For an event, don't we at least need the URI in order to download the  JSON to display transcript content?

Reply:
But that isn't needed for the cards. We have to be careful of _too_ much duplication of data. The event collection is meant for yes, event details, but also the _minimal amount of details needed to query and display event cards_ not the entire event route.

At least from my understanding

Reply:
On the event route, querying transcripts that match `event_id` and sorting by `confidence` is easier than continually updating the event collection every time we update a transcript version (if we ever do)

Reply:
I see. I thought `event` was supposed to contain everything needed for the event route.
I'm OK with `event` containing only the data needed for event cards. In that case, I agree with removing lines 20-22.